.gitallowed

```diff
@@ -0,0 +1,2 @@
+(AIDA12345667789ABCEF)

```

Comment:
Why is this needed?
Otherwise all good

Reply:
This is the way we communicate to our git-secrets plugin that these credentials are fake (safe to reveal) and the plugin shouldn't be worried about these "secrets".

Reply:
But are those used anywhere in here? Can't find it

Reply:
In `test_transformation_full.py`